### ISSUE TITLE
Error messages for JAWS/RIGB

### ISSUE THREAD

**Issue by czender** [jaws/jaws#134]
Please install user-friendly error messages for the most common RIGB errors. I assume these include:
1. not downloading the RIGB data. 2. downloading the RIGB data but the station in question does not have required radiation sensors. 3. No clear-sky dies could be found.
User-friendly means a diagnostic message and a workaround in plain English. Equivalent to the NCO
"ERROR" and "HINT" messages. The error in #133 is what made me think of this.

**Comment by ajcse1** [comment]
Done in v0.7.6
